src/main/java/com/darkyen/tproll/logfunctions/FileLogFunction.java

```diff
@@ -89,6 +93,17 @@ public void log(String name, long time, byte level, Marker marker, CharSequence
         }
     }
 
+    private void appendMarker(StringBuilder sb, Marker marker, boolean startWithSpace) {
+        if (marker instanceof RenderedMarker) {
+            if (startWithSpace) sb.append(' ');
+            sb.append("| ");
+            sb.append(marker.getName());
+        }
+        for (Iterator<Marker> it = marker.iterator(); it.hasNext(); ) {
+            appendMarker(sb, it.next(), true);
+        }
+    }
+
```

Comment:
I'd make this static, move this into `RenderedMarker` and add a flag for colors so that we don't have to maintain two variants.